Auto coding

SethMK opened the pull request.

SethMK committed:
Auto_coding

Adding auto-coding documentation

SethMK committed:
Change of order

Added change of orders of pages to fit Auto Coding page after Communication Review

peterhallerREL commented on docs/user_guide/review/auto_coding.md:
```diff
@@ -0,0 +1,43 @@
+
+From this moment when reviewing communication under given layout given field will be automatically popoulated with the provided value.
+
+Supprted field types are limited to (Single Choice, Multiple Choice, Fixed Text Length, Long Text Length, Yes/No, Whole Number, Decimal, Currency)
```
Can you merge these two info items into one info box with two bullets.

peterhallerREL commented:
@SethMK Couple pieces of feedback:

1. This information should be within the "Administrator Guide" section. An admin will configure these auto coding, not an end user. I would create a new page called "Review" within the Administrator Guide section under the "Alerting" tab.
2. On the "Coding Communications" tab within the User Guide section I would add a quick blurb about default coding decisions can be specified and then link to the new page in the administrator guide.
3. I added one more inline comment on the actual changed files.

Feel free to update this PR or make a new one and cancel this one.

SethMK committed:
Changes based on review

Updated based on feedback. Moved to Admin guide, added refference, merged info box

peterhallerREL committed:
Update coding_communications.md

peterhallerREL committed:
Update auto_coding.md

peterhallerREL committed:
Update index.md

peterhallerREL committed:
Update case_management.md

peterhallerREL committed:
Update contacting_monitored_individual.md

peterhallerREL committed:
Update index.md
